app/js/views/main.js

```diff
@@ -104,6 +104,29 @@ var mainViewTemplate =
 
     --checkbox-border-color: var(--background-minus-minus);
   }
+
+  fxos-customizer {
```

Comment:
You want me to move all of it, even the element positioning, z-index and transition stuff? That would make the customizer component very specific. These seem like view-level details rather that component level detail. Though if the component is app-specific rather than shared, then maybe it makes sense to put really specific CSS in it.

I can see that it might make sense to set the background color in the element, though we should probably use one of the variables defined in this file rather than hard-coding the color.